newIDE/app/src/UI/Dialog.js

```diff
@@ -64,6 +67,20 @@ const dialogActionPadding = 24;
 // Mobile.
 const dialogSmallPadding = 8;
 
+const getDefaultMaxWidthFromSize = (windowWidth: WidthType) => {
+  switch (windowWidth) {
+    case 'small':
+      return false;
```

Comment:
`return 'sm'` ?

Reply:
Could you type the return value of the function to avoid this in the future?

Reply:
Actually, I believe it's the right value, it's to say "on small device, do not limit the size, it can be full screen"
I'll see if I can make this clearer